hypothesis-python/docs/details.rst

```diff
@@ -694,3 +694,47 @@ on working with markers <pytest:mark examples>`.
 .. note::
     Pytest will load the plugin automatically if Hypothesis is installed.
     You don't need to do anything at all to use it.
+
+
+.. _fuzz_one_input:
+
+-------------------------
+Use with external fuzzers
+-------------------------
+
+Sometimes, you might want to point a traditional fuzzer such as
+`python-afl <https://github.com/jwilk/python-afl>`__ or :pypi:`pythonfuzz`
+at your code. Wouldn't it be nice if you could use any of your
+:func:`@given <hypothesis.given>` tests as fuzz targets?
+
+.. code:: python
+
+    @given(st.text())
+    def test_foo(s):
+        ...
+
+
+    # This is a traditional fuzz target - call it with a bytestring,
+    # or a binary IO object, and it runs the test once.
+    fuzz_target = test_foo.hypothesis.fuzz_one_input
+
+    # For example:
+    fuzz_target(b"\x00\x00\x00\x00\x00\x00\x00\x00")
```

Comment:
We should probably mention somewhere that the actual format here is not one that is stable between Hypothesis versions and may change arbitrarily.